app/src/components/posts/VideoPlayer.vue

```diff
@@ -1,6 +1,9 @@
 <script setup lang="ts">
 import type { Post, Tag } from "@/types";
 import videojs from "video.js";
+import "videojs-mobile-ui";
+// TODO: fix typings
+import type mobileUi from "videojs-mobile-ui";
```

Comment:
It could very well be that this doesn't work because the version of the typings is 0.8 while the package is 1.1, it seems the typings haven't been updated in a long time. We could either just remove it for now, or maybe commit a new change to the [types repo](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/videojs-mobile-ui).